oldwinapi/windows.nim

```diff
@@ -9149,8 +9153,8 @@ type
     rclBounds*: RECTL
     nPolys*: DWORD
     cpts*: DWORD
-    aPolyCounts*: array[0..0, DWORD]
-    apts*: array[0..0, POINTS]
+    aPolyCounts*: UncheckedArray[DWORD]
+    apts*: UncheckedArray[POINTS]

```

Comment:
This cannot be correct either. Only the last field in an object can be an UncheckedArray.

Reply:
`apts` and `abTypes` are just pointers to "flexible arrays", so I think this is fine.
https://msdn.microsoft.com/en-us/library/windows/desktop/dd162560(v=vs.85).aspx

Reply:
Very strange, but ok.